src/lib/journald.ts

```diff
@@ -45,3 +45,11 @@ export function spawnJournalctl(opts: {
 
 	return journald;
 }
+
+export function setJournalRotationSize(size: string)  => {
+	execSync('sed', [
```

Comment:
Why use `sed`? It would be better to use typescript, rather than assuming people are happy with sed.

Reply:
Just re-pushed with a simple `readFileSync/writeFileSync`

(however the filesystem itself being ro is a problem still)